.changelog/5537.bugfix.md

```diff
@@ -0,0 +1 @@
+Include the mode of the node in control status output
```

Comment:
Since cbor decoding in grpc allows extra fields, it might be ok to not consider this change breaking